viam_cartographer_test.go

```diff
@@ -129,7 +145,7 @@ func TestNew(t *testing.T) {
 		defer fsCleanupFunc()
 
 		attrCfg := &vcConfig.Config{
-			Sensors:       []string{"good_lidar"},
```

Comment:
this merges `Successful creation of cartographer slam service in localization mode` and `Successful creation of cartographer slam service in localization mode when feature flag enabled`